packages-ts/gauntlet-cosmos-contracts/networks/.env.local

```diff
@@ -0,0 +1,6 @@
+NODE_URL=http://localhost:26657
+DEFAULT_GAS_PRICE="0.025ucosm"
```

Comment:
should this be a decimal without the token denomination rather than a string, as in line with the other env files?